index.html

```diff
@@ -135,6 +143,86 @@ <h2>Project name goes here</h2>
       </div>
     </section>
 
+    <section class="desktop-work">
+      <h2 class="desktop-projects">Projects</h2>
+      <div class="my-first-project">
+        <div class="project-overlay"></div>
+        <h2>Project name goes here</h2>
+        <ul>
+          <li>HTML/CSS</li>
```

Comment:
- [ ] Based on the recommendation of the previous reviewer to stick to the design, I would implore you to give space to the pointed aspect in the images below.

### Figma
![image](https://user-images.githubusercontent.com/86200356/236645552-6aac4c80-d837-4a64-8ad3-15b6b478bcfd.png)

### Your design
![image](https://user-images.githubusercontent.com/86200356/236645580-198d57b5-bb0b-4ccc-8aea-5bb38de96af0.png)